### ISSUE TITLE
Latest acorn breaks parsing 

### ISSUE THREAD

**Issue by jknight12882** [zloirock/core-js#277]
```
ERROR in ./~/core-js/library/modules/_export.js
Module parse failed: /Users/.../node_modules/core-js/library/modules/_export.js Parenthesized pattern (46:6)
You may need an appropriate loader to handle this file type.
|     // export proto methods to core.%CONSTRUCTOR%.methods.%NAME%
|     if(IS_PROTO){
|       (exports.virtual || (exports.virtual = {}))[key] = out;
|       // export proto methods to core.%CONSTRUCTOR%.prototype.%NAME%
|       if(type & $export.R && expProto && !expProto[key])hide(expProto, key, out);
 @ ./~/core-js/library/modules/es6.object.assign.js 2:14-34
 @ ./~/core-js/library/fn/object/assign.js
 @ ./~/babel-runtime/core-js/object/assign.js
 @ ./~/babel-runtime/helpers/extends.js
```

Probably related to https://github.com/ternjs/acorn/commit/ba60eeae55e58f89dead4b8f639460dcd5d56a29

**Comment by loganfsmyth** [comment]
Isn't this an Acorn bug? There are no patterns in this code, so there's definitely no parens around patterns.

**Comment by jknight12882** [comment]
Could be, looks like there's a bunch of issues being reported for the `acorn@4.0.5` release

**Comment by loganfsmyth** [comment]
I'll file it over there.

**Comment by loganfsmyth** [comment]
Filed^

**Comment by zloirock** [comment]
Seems it's fixed in `acorn@4.0.8`.
